scripts/build.mjs

```diff
@@ -70,7 +80,19 @@ class Builder {
     // environment variable `__COMPAT_LAYER=RunAsInvoker` to force the installer
     // to run as the existing user.
     const env = { ...process.env, __COMPAT_LAYER: 'RunAsInvoker' };
+    const releaseBuildIndex = args.indexOf('--release');
```

Comment:
So if `--release` is given, we use the whole `git describe` output, otherwise we only match the previous tag?  That seems odd, since on actual releases, the `git describe` output would just be the tag (since `git describe` without `--long` will only emit the tag on an actual tagged commit).  Maybe call this `--long-version` or something instead?

Is there a reason we would _not_ want the full `git describe` output? When building releases, since the `-0-abcdef` part wouldn't be there (just the tag), that should still be fine?

Reply:
It's the other way. If `--release` is given we use only the version part of the described tag. If we're running in dev mode or making a local build, we use the full tag.